authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -0,0 +1,95 @@
+[id="about-cloud-credential-operator"]
```

Comment:
This overview is pulled from the old location of the CCO info, with the various modes split off into their own sub-areas.

Reply:
Preview: https://deploy-preview-29545--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html